mozilla_django_oidc_db/models.py

```diff
@@ -31,7 +40,63 @@ def get_claim_mapping() -> Dict[str, str]:
     }
 
 
-class OpenIDConnectConfig(SingletonModel):
+class CachingMixin:
+    @classmethod
+    def clear_cache(cls):
+        cache_name = getattr(
+            settings, "OIDC_CACHE", oidc_settings.MOZILLA_DJANGO_OIDC_DB_CACHE
+        )
+        if cache_name:
+            cache = get_cache(cache_name)
+            cache_key = cls.get_cache_key()
+            cache.delete(cache_key)
+
+    def set_to_cache(self):
+        cache_name = getattr(
+            settings,
+            "MOZILLA_DJANGO_OIDC_DB_CACHE",
+            oidc_settings.MOZILLA_DJANGO_OIDC_DB_CACHE,
+        )
+        if not cache_name:
+            return None
+        cache = get_cache(cache_name)
+        cache_key = self.get_cache_key()
+        timeout = getattr(
+            settings,
+            "MOZILLA_DJANGO_OIDC_DB_CACHE_TIMEOUT",
+            oidc_settings.MOZILLA_DJANGO_OIDC_DB_CACHE_TIMEOUT,
+        )
+        cache.set(cache_key, self, timeout)
+
+    @classmethod
+    def get_cache_key(cls):
+        prefix = getattr(
+            settings,
+            "MOZILLA_DJANGO_OIDC_DB_PREFIX",
+            cls.custom_oidc_db_prefix,
+        )
```

Comment:
Shouldn't this check the class property first and then fall back to the (global) setting? Now if you define the setting, it looks like you can't override the prefix anymore on a (sub)class basis.

Reply:
you're right, I changed it so now the order is:
1. classproperty `custom_oidc_db_prefix`
2. `settings.MOZILLA_DJANGO_OIDC_DB_PREFIX`
3. default from library `oidc_settings.MOZILLA_DJANGO_OIDC_DB_PREFIX`